src/tests/ports/catalog-queries.spec.ts

```diff
@@ -388,4 +389,84 @@ test('catalog utils', function () {
       expect(query.values).toStrictEqual([limit, offset])
     })
   })
+
+  describe('when there is a prince range applied', () => {
+    let filters: CatalogFilters
+    let minPrice: string
+    let maxPrice: string
+    describe('and there is only min price applied', () => {
+      beforeEach(() => {
+        minPrice = '10'
+        filters = {
+          minPrice,
+        }
+      })
+      it('should apply the range to the orders table JOIN', () => {
+        expect(
+          getCollectionsItemsCatalogQuery('aSchemaVersion', filters).text
```

Comment:
WDYT about moving this into a variable that is set after the `filters` in the `beforeEach` so it can be used later in the different `expect` statements?

Reply:
good call, updated!